torch/csrc/autograd/profiler_kineto.cpp

```diff
@@ -102,12 +107,16 @@ struct MetadataBase {
 struct AddTensorboardFields : public MetadataBase {
   AddTensorboardFields(
       const std::shared_ptr<Result>& result,
-      KinetoEvent& kineto_event)
+      KinetoEvent& kineto_event,
+      const bool no_callstack_trace)
       : MetadataBase(result) {
     result->visit(*this);
     const auto module_hierarchy = kineto_event.moduleHierarchy();
     addMetadata("Module Hierarchy", stacksToStr(module_hierarchy.vec(), "."));
-    addMetadata("Call stack", stacksToStr(kineto_event.stack().vec(), ";"));
+    addMetadata(
```

Comment:
This would also drop call stack from JIT, which we want to preserve. (Since the end goal is to flip a switch.)

Reply:
I changed to drop call stack info is KinetoEvent constructor, where `python_stack_` is built, and changes are working fine in my local test.